README.md

```diff
@@ -253,6 +253,29 @@ Promise.all([builder.trace('app/first.js'), builder.trace('app/second.js')])
 });
 ```
 
+### Using the compiler cache
```

Comment:
Did you mean "to" rather than "not to" I'm not sure which way to make sense of that sentence.